package.json

```diff
@@ -121,18 +121,19 @@
     "@qlik/prettier-config": "0.4.5",
     "@rollup/plugin-typescript": "11.1.5",
     "@scriptappy/from-jsdoc": "0.19.0",
-    "@testing-library/jest-dom": "6.1.6",
+    "@testing-library/jest-dom": "6.2.0",
     "@testing-library/react": "14.1.2",
     "@testing-library/react-hooks": "8.0.1",
     "@testing-library/user-event": "14.5.2",
     "@types/jest": "29.5.11",
-    "@types/react": "18.2.46",
+    "@types/react": "18.2.47",
     "@types/react-window": "1.8.8",
     "@typescript-eslint/eslint-plugin": "5.62.0",
     "@typescript-eslint/parser": "5.62.0",
     "conventional-changelog-cli": "4.1.0",
     "eslint": "8.56.0",
-    "eslint-plugin-jest": "27.6.0",
+    "eslint-plugin-jest": "27.6.1",
+    "eslint-plugin-playwright": "^0.21.0",
```

Comment:
Pin version

Reply:
On a second note. There should be no need to add it as a dependency as it will get installed via `@qlik/eslint-config`.

No sure why it complained for you. Tried to delete node_modules and then install again?

Reply:
yeah I thought it shouldn't be needed too. I'll try removing node_modules

Reply:
Seems to get the same result. Plus I got it on circle, so there is must be a new node_modules there every time in the install step

Reply:
After some more digging the modules ends up in `node_modules/@qlik/eslint-config/node_modules/eslint-plugin-playwright`. So it's installed but not in root

Reply:
It could be a bug in `@qlik/eslint-config` as the package should be set as peerDep if the consumer needs to install it. Otherwise `@qlik/eslint-config` might expect version X of some dependency but we have it in our depDeps with version Y. Possibly causing failures when linting.

Reply:
As we have seen with `typescript-eslint/eslint-plugin` and `@typescript-eslint/parser`

Reply:
But for know just add it as as devDeps

Reply:
FYI: when using `pnpm` `eslint-plugin-playwright` get installed (symlinked) in root. At least in `sn-action-button`. Maybe they are expecting consumers to use pnpm?

Reply:
You get this warning when installing with `yarn`

> warning @qlik/prettier-config@0.4.5: The engine "pnpm" appears to be invalid.

Reply:
ah ok. I'll add the plugin for now and we can remove it when we move to pnpm

Reply:
I know why it worked with `pnpm` now. It by default hoist all eslint plugins to root node_modules.